cmake/FindMetaExamples.cmake

```diff
@@ -37,13 +36,6 @@ function(get_excluded_meta_examples)
 			)
 	ENDIF()
 
-	IF(NOT HAVE_LAPACK)
-		LIST(APPEND EXCLUDED_META_EXAMPLES
```

Comment:
please keep the block, so we can add things there that pop up in the future, just don't append anything